src/discordEvents/messageCreate.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'messageCreate',
+    async execute(client, message) {
+        let instance = client.readInstanceFile(message.guild.id);
+        let channel = instance.channelId.ingamechat
+        if (message.channelId != channel) return;
+        if (message.author.bot) return;
+        let rustplus = client.rustplusInstances[message.guild.id];
+        rustplus.sendTeamMessage(`Discord -> Game | ${message.author.username}: ${message.cleanContent}`)
```

Comment:
Don't think we need Discord _> Game | part, just have the discord name with colon message